frontend/src/app/shared/confirmation-modal/confirmation-modal.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input } from "@angular/core";
+import { NgbActiveModal } from "@ng-bootstrap/ng-bootstrap";
+
+@Component({
+    selector: "lc-confirmation-modal",
+    templateUrl: "./confirmation-modal.component.html",
+    styleUrls: ["./confirmation-modal.component.scss"],
+})
+export class ConfirmationModalComponent {
+    @Input() public title = "Lettercraft";
+    @Input() public message = "Are you sure you want to continue?";
+
+    constructor(private activeModal: NgbActiveModal) {}
+
+    public confirm(): void {
+        this.activeModal.close(true);
+    }
+
+    public cancel(): void {
+        this.activeModal.close(false);
+    }
```

Comment:
The reason is more convenience than anything else. Calling `dismiss()` rejects the Promise that is pending while the modal is active, so we have to `.catch()` it if we want to prevent an error. It _is_ the nicest way to do it, so I'll make sure it's implemented as such.